front/app/components/Form/Components/Controls/controlUtils.tsx

```diff
@@ -26,10 +26,10 @@ export const getOptions = (schema, fieldType: 'single' | 'multi') => {
     );
   } else {
     return (
-      schema?.oneOf
```

Comment:
If I do a global search for `:oneOf`, I still see a couple of BE services using it. Are we not breaking those by completely replacing this check?

Reply:
As long as they are just regular JSON schemas and not a JSON Forms schema, it shouldn't be an issue. I'm going to do a functional test now though to make sure all instances where JSON Forms are used are still working correctly :)

Reply:
Ah shoot - you're right. I didn't realize this utility is being used outside JSON forms. I'll make a quick tweak :+1: